src/components/ReportWelcomeText.tsx

```diff
@@ -112,7 +118,38 @@ function ReportWelcomeText({report, policy, personalDetails}: ReportWelcomeTextP
                             <Text>{welcomeMessage.phrase3}</Text>
                         </Text>
                     ))}
+                {isInvoiceRoom &&
+                    !isArchivedRoom &&
+                    (welcomeMessage?.messageHtml ? (
+                        <PressableWithoutFeedback
+                            onPress={() => {
+                                if (!canEditPolicyDescription) {
+                                    return;
+                                }
+                                Navigation.navigate(ROUTES.WORKSPACE_PROFILE_DESCRIPTION.getRoute(policy?.id ?? '-1'));
```

Comment:
Clicking description is navigating to workspace description, even though this is description of invoice room. https://github.com/Expensify/App/issues/51976